dev/TabView/TabView_themeresources.xaml

```diff
@@ -32,21 +47,51 @@
+            <StaticResource x:Key="TabViewItemIconForegroundDisabled"        ResourceKey="SystemControlForegroundBaseMediumBrush" />
+            <StaticResource x:Key="TabViewButtonBackground"                  ResourceKey="SystemControlTransparentBrush" />
+            <StaticResource x:Key="TabViewButtonBackgroundPressed"           ResourceKey="SystemControlBackgroundListMediumBrush" />
+            <StaticResource x:Key="TabViewButtonBackgroundPointerOver"       ResourceKey="SystemControlBackgroundListLowBrush" />
+            <StaticResource x:Key="TabViewButtonBackgroundDisabled"          ResourceKey="SystemControlTransparentBrush" />
+            <StaticResource x:Key="TabViewButtonBackgroundActiveTab"         ResourceKey="SystemControlTransparentBrush" />
+            <StaticResource x:Key="TabViewButtonForeground"                  ResourceKey="SystemControlBackgroundBaseMediumBrush" />
+            <StaticResource x:Key="TabViewButtonForegroundPressed"           ResourceKey="SystemControlForegroundBaseMediumHighBrush" />
+            <StaticResource x:Key="TabViewButtonForegroundPointerOver"       ResourceKey="SystemControlForegroundBaseMediumHighBrush" />
+            <StaticResource x:Key="TabViewButtonForegroundDisabled"          ResourceKey="SystemControlDisabledBaseMediumLowBrush" />
+            <StaticResource x:Key="TabViewButtonForegroundActiveTab"         ResourceKey="SystemControlBackgroundBaseMediumBrush" />
+            <StaticResource x:Key="TabViewScrollButtonBackground"            ResourceKey="SystemControlTransparentBrush" />
             <StaticResource x:Key="TabViewItemSeparator"                     ResourceKey="SystemControlForegroundBaseLowBrush" />
         </ResourceDictionary>
 
         <ResourceDictionary x:Key="HighContrast">
-            <StaticResource x:Key="TabViewBackground"                        ResourceKey="SystemControlBackgroundBaseLowBrush" />
-            <StaticResource x:Key="TabViewItemHeaderBackground"              ResourceKey="SystemControlTransparentBrush" />
-            <StaticResource x:Key="TabViewItemHeaderBackgroundSelected"      ResourceKey="SystemControlHighlightBaseHighBrush" />
-            <StaticResource x:Key="TabViewItemHeaderBackgroundPointerOver"   ResourceKey="SystemAltMediumLowColor" />
-            <StaticResource x:Key="TabViewItemHeaderBackgroundPressed"       ResourceKey="SystemAltMediumColor" />
+            <StaticResource x:Key="TabViewBackground"                        ResourceKey="SystemControlBackgroundChromeMediumLowBrush" />
+            <StaticResource x:Key="TabViewItemHeaderBackground"              ResourceKey="SystemControlBackgroundBaseLowBrush" />
+            <StaticResource x:Key="TabViewItemHeaderBackgroundSelected"      ResourceKey="SystemControlHighlightChromeHighBrush" />
+            <StaticResource x:Key="TabViewItemHeaderBackgroundPointerOver"   ResourceKey="SystemControlHighlightChromeHighBrush" />
```

Comment:
I'm not a fan of Edge (chromium) approach of having the selected tab look the same as the tab being hovered over. I think Edge (spartan) got the accessibility right in that the hovered tab gains a border but not a full background change.

Reply:
That's a fair point, and I'm also slightly uncomfortable with the loss of information of the selected-tab looking the same as a hovered-tab.  However, the border *is* exposed programmatically, so a developer could set one too.  It'll take some finesse, I think, to insert a border without messing up apps that also try to set a border.

As to what this should look like, I'll present a few options.  In each of these mockups, the first tab is active, the second tab is hovered, and the third is idle.

Border | Mock-up
--- | ---
4px all sides | ![image](https://user-images.githubusercontent.com/10259764/71772063-5110a080-2ef9-11ea-80bb-3e29c1657381.png)
2px all sides | ![image](https://user-images.githubusercontent.com/10259764/71771960-f296f280-2ef7-11ea-8384-3db6e75eb3b2.png)
4px top | ![image](https://user-images.githubusercontent.com/10259764/71771973-1ce8b000-2ef8-11ea-858e-0924993864a5.png)
4px top and sides | ![image](https://user-images.githubusercontent.com/10259764/71771988-43a6e680-2ef8-11ea-8a27-3b03abc2dddb.png)
4px bottom | ![image](https://user-images.githubusercontent.com/10259764/71772032-d5aeef00-2ef8-11ea-9c62-408beccb7d0f.png)

I'm no UI designer, so I don't hold my opinion with any great fondness, but I tend to think that the "4px all sides" option looks the most consistent with the rest of the OS, although the "4px bottom" option is actually kind of cool to interact with.

Reply:
@stmoy, @chigy any thoughts? My personal preference is the "2px all sides" option, which feels consistent with other controls styling and also with Edge (Spartan) UX.

Reply:
+1 to 2px on all sides. 4px on all sides seems overly chunky to me, and the other options seem inconsistent with the platform all-up.

Reply:
2px border thickness is being used for the Form controls when focused, so it seems sensible to use that here.  perhaps the 4px top border with 2px on the sides could be a good middle ground

Reply:
Agree with the 2px all sides feeling best.